<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SortAlgorithms</class>
 <widget class="QMainWindow" name="SortAlgorithms">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SortAlgorithms</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listW_RawInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>171</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>List of Numbers</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txEd_SortFootprint">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>471</width>
      <height>181</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort Footprint</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Randomize">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Randomize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Add">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Remove">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>140</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Method</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="rdBtn_InsertionSort">
       <property name="text">
        <string>Insertion Sort</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rdBtn_HeapSort">
       <property name="text">
        <string>Heap Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rdBtn_QuickSort">
       <property name="text">
        <string>Quick Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rdBtn_MergeSort">
       <property name="text">
        <string>Merge Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rdBtn_SelectSort">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Selection Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rdBtn_BubbleSort">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Bubble Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_DoSort">
       <property name="text">
        <string>Do Sort</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Num. of Entry</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spBx_NumOfEntry">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ClearFootprint">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>543</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
